repose-aggregator/components/filters/saml-policy-translation-filter/src/main/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilter.scala

```diff
@@ -493,7 +495,7 @@ class SamlPolicyTranslationFilter @Inject()(configurationService: ConfigurationS
     private var initialized = false
 
     override def configurationUpdated(configurationObject: SystemModel): Unit = {
-      sendTraceHeader = Option(configurationObject.getTracingHeader).map(_.isEnabled).getOrElse(true)
+      sendTraceHeader = Option(configurationObject.getTracingHeader).forall(_.isEnabled)
```

Comment:
I was clearing all the items in the right hand trough and this is one the IntelliJ static analysis found.